examples/security-insights-minimal-sample.yml

```
@@ -13,6 +13,7 @@ project-lifecycle:
   - https://github.com/scovetta
 contribution-policy:
   accepts-pull-requests: true
+  accepts-bot-pull-requests: false
```

```suggestion
  accepts-bot-pull-requests: true
```

Or remove it and go with the default

---

I change to `true` for now.